### PR TITLE
Personal Portfolio 

### PR DESCRIPTION
**Pull request by anishajain22**
Updated readme to customize it.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -26,7 +29,15 @@
...
-    response.getWriter().println("<h1>Hello world!</h1>");
+    
+    List<String> list = new ArrayList<String>();
+    list.add("Coffee");
+    list.add("Cat");
+    list.add("Chocolate");
+
+    String json=new Gson().toJson(list);
+    
+    response.setContentType("application/json;");
```
Comment (yashjaing) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:40]: I think you dont need a ";" after json. I'd also suggest using string constants (private static final String APPLICATION_TYPE = "application/json").

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -26,7 +29,15 @@
...
+    response.getWriter().println(json);
```
Comment (yashjaing) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:41]: I think you need to close the writer as well.
Check if try with resources https://www.baeldung.com/java-try-with-resources can help you do that automatically!

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -15,14 +15,44 @@
...
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
+  const facts =
+      ['I like sketching a lot!', 'I am a foodie<3', 'I have a massive sweet tooth!', 'A big movie buff:)'];
 
   // Pick a random greeting.
```
Comment (yashjaing) [inline portfolio/src/main/webapp/script.js:22]: Replace greeting with fact :P

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -15,14 +15,44 @@
...
+        const statsListElement = document.getElementById('comments-container');
+    statsListElement.innerHTML = '';
+    statsListElement.appendChild(
+        createListElement(json[0]));
+    statsListElement.appendChild(
+        createListElement(json[1]));
+    statsListElement.appendChild(
+        createListElement(json[2]));
+
+    console.log(json);
```
Comment (yashjaing) [inline portfolio/src/main/webapp/script.js:47]: please remove

Comment (anishajain22) [reply portfolio/src/main/webapp/script.js:47]: This was just for testing as in one of the tasks of the week. Will change for final code.

**Pull request merged**